app/BUILD.bazel

```diff
@@ -555,6 +554,7 @@ kt_android_library(
     deps = [
         ":dagger",
         "//model",
+        artifact("nl.dionsegijn:konfetti:1.2.5"),
```

Comment:
I was starting at the highest-level library to see if it would build and forgot to move it -- it doesn't build when I put it in the app library though this makes the most sense to me.
Moved to databinding_resources

Reply:
Ah, databinding_resources actually makes a lot of sense since it contains all of our databinding-enabled layouts (of which the one we're using Konfetti in is one). There is also an "indirect" dep coming in when we build the app module, and turning on strict deps would require also adding this artifact to the app module. I think we can address strict deps later--what you have here is definitely correct & sufficient.